### ISSUE TITLE
What's what now?

### ISSUE THREAD

**Issue by fireheadman** [zynine-/HDHRViewerV2.bundle#24]
Have seen a slew of recent updates to this plugin.
Noticed there is no longer a place to select logos per your website screenshots.
http://hdhrviewer.zynine.net/hdhrviewerv2-initial-setup/

....was curious when the documentation on your website will catch up to all the updates?
Should we still be using zap2it for channel scraping/logos?
Should we be making our own logos?

....please advise.
And liking that this plugin is still active!

-Ben

Screenshot from latest plugin:
Notice there is not a "Channel Logo Match by Channel - Number"
![image](https://cloud.githubusercontent.com/assets/19847195/19628613/3ce2ad56-991f-11e6-98a2-3a23ed675ee8.png)

Screenshot from your website:
![image](https://cloud.githubusercontent.com/assets/19847195/19628619/55539382-991f-11e6-877e-195a53820fe7.png)

**Comment by zynine-** [comment]
Plug-in is designed to give you some flexibility. I'm slowly moving stuffs to the newer spot: https://github.com/zynine-/HDHRViewerV2.bundle/wiki 

For beginners, you can always set:
**HDHomeRun IP** = auto
**XMLTV Mode** = hdhomerun

Otherwise, refer to:
https://github.com/zynine-/HDHRViewerV2.bundle/wiki/Channel-Guide-(EPG)
https://github.com/zynine-/HDHRViewerV2.bundle/wiki/Channel-Icons-Logo

**Comment by fireheadman** [comment]
Setting XMLTV Mode to hdhomerun was the key.   Did this at the same time you were writing the reply, I also deleted all the fields I previously filled out, saved, then restarted plex.

After logging back in, I had to select either "poster" or "details" view to get the icons to show up, which I will say look VERY GOOD!   This is much much better than having to deal with setting up a zap2it account then create a health script and run via crontab just to pull info.

I am curious, how often does this plug update channel metadata for program synopsis? (ie, show playing and details of show)
